src/org.ts

```diff
@@ -799,43 +799,42 @@ export class Org extends AsyncCreatable<Org.Options> {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   private async removeUsers(throwWhenRemoveFails: boolean): Promise<void> {
```

Comment:
we could remove the parameter since it's not used and private

Reply:
I almost did, and then left it for fear of that thing TS does where it says Org !== Org if private methods have different signatures.

v3 might be a better opportunity.